scripts/post_process_dispersy_experiment.sh

```diff
@@ -101,7 +101,7 @@ export XMAX
 graph_data.sh
 
 #Step 6: Check historical data for large deviation
-post_process_historical_values.py
+#post_process_historical_values.py
```

Comment:
Maybe use a flag in the script to turn historical_values on or off (or an environment variable or something). Commenting it out completely "hurts" the AllChannel experiment(s).